src/test/java/tools/jackson/databind/ext/javatime/deser/OneBasedMonthDeserTest.java

```diff
@@ -177,12 +177,13 @@ public void testDeserializeFromEmptyString() throws Exception
         assertNull(m);
 
         // But coercion from empty String not enabled for Enums by default:
-        try {
-            mapper.readerFor(Month.class).readValue("\"\"");
-            fail("Should not pass");
-        } catch (MismatchedInputException e) {
-            verifyException(e, "Cannot coerce empty String");
-        }
+        // TODO : Figure out how to make this pass.
+//        try {
+//            Month result = mapper.readerFor(Month.class).readValue("\"\"");
+//            fail("Should not pass");
+//        } catch (MismatchedInputException e) {
+//            verifyException(e, "Cannot coerce empty String");
+//        }
```

Comment:
Current implementation returns `null` with or without below CoercionSetting 🤔.

Reply:
This may be way to go as MonthDayDeser... also behaves the same

Reply:
I figured it out -- is bit complicated but rules changed when `LogicalType` became `DateTime` instead of `Enum` (technically is an Enum but logically DateTime (more refined)).

So I changed test a bit.